drivers/sensor/sgp40/sgp40.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SGP40_SGP40_H_
+#define ZEPHYR_DRIVERS_SENSOR_SGP40_SGP40_H_
+
+#include <device.h>
+#include <kernel.h>
+#include <drivers/gpio.h>
```

Comment:
why are these 2 headers needed by this header?

Reply:
probably because of cargo-cult programming. i am new to zephyr..
i have removed them.